R/step-subset-filter.R

```diff
@@ -32,6 +32,21 @@ filter.dtplyr_step <- function(.data, ..., .preserve = FALSE) {
   step_subset_i(.data, i)
 }
 
+filter_by_lgl_col <- function(dots) {
+  if (length(dots) > 1) {
+    return(FALSE)
+  }
+
+  dot <- dots[[1]]
+
+  if (is_symbol(dot)) {
+    return(TRUE)
+  }
```

Comment:
This is a super small comment, but I've been thinking about this lately — I'd suggest more clearly making this a "paragraph" of code by removing the blank line on line 41. I think that it makes these two statements hang together better as a related thought.

Reply:
Thinking about it I agree 👍